metrics/fidelity/fidelity.py

```diff
@@ -0,0 +1,145 @@
+import logging
+
+import numpy as np
+
+import metrics.fidelity.utility as utility
+import datasets
+
+
+logger = logging.getLogger(__name__)
+
+_CITATION = """
+citation is under construction 
+@misc{carton2020evaluating,
+      title={Evaluating and Characterizing Human Rationales}, 
+      author={Samuel Carton and Anirudh Rathore and Chenhao Tan},
+      year={2020},
+      eprint={2010.04736},
+      archivePrefix={arXiv},
+      primaryClass={cs.CL}
+}
+"""
+
+_DESCRIPTION = """
+description is under construction
+sufficiency(x, y_hat, alpha) = 1 - max(0, P(y_hat/x) - P(y_hat/x, alpha))
+comprehensiveness(x, y_hat, alpha) = max(0, P(y_hat/x) - P(y_hat/x, alpha))
+null difference = max(0, P(y_hat/x) - P(y_hat/x, 0))
```

Comment:
add definitions for normalized sufficiency, and normalized comprehensivenss?

Reply:
Yeah, each of these arguments needs to be explained.